eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
+		"react/boolean-prop-naming": "error",
+		"react/button-has-type": "error",
+		"react/default-props-match-prop-types": "error",
+		"react/jsx-boolean-value": ["error", "never"],
+		"react/jsx-child-element-spacing": "error",
+		"react/jsx-closing-bracket-location": "error",
+		"react/jsx-closing-tag-location": "error",
+		"react/jsx-curly-brace-presence": "error",
+		"react/jsx-curly-spacing": ["error",  {
+			"when": "always",
+			"children": true
+		}],
+		"react/jsx-equals-spacing": ["error", "never"],
+		"react/jsx-filename-extension": "error",
+		"react/jsx-first-prop-new-line": "error",
+		"react/jsx-fragments": ["error", "element"],
```

Comment:
Is there a special reason to enforce the `<React.Fragment>` instead of `<>`?

Reply:
The docgen :(((

Reply:
The idea is to force `<>` when and if fixing this issue with the docgen